mrd-storage-server/meta.yaml

```diff
@@ -0,0 +1,20 @@
+package:
+  name: mrd-storage-server
+  version: 1.5.0
+
+source:
+  git_rev: v0.0.7
+  git_url: https://github.com/ismrmrd/mrd-storage-server
+
+requirements:
+  build:
+    - go-cgo=1.17.6
+
+  run: {}
+
+test:
+  requires:
+    - curl
+
+about:
```

Comment:
I know I'm raising the bar, but I think we should fill in the metadata as best as we can when we make new recipes or tweak the existing ones. I have just added some metadata for the ISMRMRD package (#11).

Reply:
Good idea. Added.